### PR TITLE
Moose PSU

### PR DESCRIPTION
**Pull request by simopier**
meta.yaml file for CHiMaD problems. Pb 1a using IA time stepper.

### CODE REVIEW

File: _data/simulations/moose_psu_1a_IA/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+benchmark:
+  id: 1a
+  version: 1
+
+metadata:
+  summary: Metadata for the Benchmark1 problems
+  timestamp: 02/20/2017
+  author: PC. Simon
+  email: pjs5523@psu.edu
+  timestamp: "Thu Jan 19 15:21:30 2017 -EST"
```
Comment (wd15) [inline _data/simulations/moose_psu_1a_IA/meta.yaml]: Not a valid time stamp, use `Wed Feb 22 15:18:26 EST 2017` for example

File: _data/simulations/moose_psu_1a_IA/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  software:
+    name: moose
+    version: git commit 2593a61 on 2017-01-18
+  hardware:
+    computing plateform: Red Hat Entreprise Linux 6 (Cluster)
```
Comment (wd15) [inline _data/simulations/moose_psu_1a_IA/meta.yaml]: `computing plateform` is not a valid key

File: _data/simulations/moose_psu_1a_IA/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    details:
+      - name: clock rate
+        values:
+          rate: 
+          unit: GHz
+      - name: processor
+        values: ""
+  implementation:
+    repo:
+      url:
```
Comment (wd15) [inline _data/simulations/moose_psu_1a_IA/meta.yaml]: Need a value in this field

File: _data/simulations/moose_psu_1a_IA/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      - name: clock rate
+        values:
+          rate: 
+          unit: GHz
+      - name: processor
+        values: ""
+  implementation:
+    repo:
+      url: 
+      version:
```
Comment (wd15) [inline _data/simulations/moose_psu_1a_IA/meta.yaml]: Need a value in this field

**Comment by wd15** [comment]
@simopier, the following changes fixes the `meta.yaml` for me, I'll let you fix it and then I'll merge and then you can fix the others pull requests

## First Change
```
-    architecture: Red Hat Entreprise Linux 6 (Cluster)
+    architecture: x86_64
```
The `architecture` field needs to be one of [these](https://github.com/usnistgov/chimad-phase-field/blob/master/_data/simulations/example/schema.yaml#L68).

## Second Change
```
  implementation:
     repo:
       url: https://github.com/simopier/CHiMaD/blob/master/H1.i
-      version: 1
+      version: "724999206"
     end_condition: Equilibrium, total energy fluctuated due to numerical noise
     details:
```
The `version` field needs to be a valid commit hash (at least 5 characters), see the [spec](https://github.com/usnistgov/chimad-phase-field/blob/master/_data/simulations/example/schema.yaml#L95). 


Everything else was good and my test site plotted the curve in the 1a chart correctly. Please do make those changes and we can move forward. Thanks!

**Comment by simopier** [comment]
Dear Daniel Wheeler,

Thank you very much for your reviews!

I try to address them as I could, let me know if I still to change some
things.

Best regards,

Pierre-Clement Simon
Graduate Research Assistant in Nuclear Engineering
The Pennsylvania State University
128 Hammond Building
University Park, PA 16802
+1 814-777-6967

**Pull request merged**